sdk/keyvault/keyvault-keys/recordings/browsers/keys_client__create_read_update_and_delete_operations_for_managed_hsm_getrandombytes/recording_supports_tracing.json

```diff
@@ -1,78 +1,78 @@
-    "x-content-type-options": "nosniff",
-    "x-frame-options": "SAMEORIGIN",
-    "x-ms-request-id": "5c066560-d529-11eb-b1e5-0022484f830d",
-    "x-ms-server-latency": "0"
-   }
+  "recordings": [
+    {
+      "method": "POST",
+      "url": "https://azure_managedhsm.managedhsm.azure.net/rng",
+      "query": {
+        "api-version": "7.3"
+      },
+      "requestBody": "",
+      "status": 401,
+      "response": "",
+      "responseHeaders": {
+        "cache-control": "no-cache",
+        "content-length": "0",
+        "content-security-policy": "default-src 'self'",
+        "content-type": "application/json; charset=utf-8",
+        "strict-transport-security": "max-age=31536000; includeSubDomains",
+        "www-authenticate": "Bearer authorization=\"https://login.microsoftonline.com/12345678-1234-1234-1234-123456789012\", resource=\"https://managedhsm.azure.net\"",
+        "x-content-type-options": "nosniff",
+        "x-frame-options": "SAMEORIGIN",
+        "x-ms-request-id": "5c066560-d529-11eb-b1e5-0022484f830d",
+        "x-ms-server-latency": "0"
+      }
+    },
+    {
+      "method": "POST",
+      "url": "https://login.microsoftonline.com/12345678-1234-1234-1234-123456789012/oauth2/v2.0/token",
+      "query": {},
+      "requestBody": "response_type=token&grant_type=client_credentials&client_id=azure_client_id&client_secret=azure_client_secret&scope=https%3A%2F%2Fsanitized%2F",
+      "status": 200,
+      "response": "{\"token_type\":\"Bearer\",\"expires_in\":86399,\"ext_expires_in\":86399,\"access_token\":\"access_token\"}",
+      "responseHeaders": {
+        "cache-control": "no-store, no-cache",
+        "content-length": "1322",
+        "content-type": "application/json; charset=utf-8",
+        "date": "Thu, 24 Jun 2021 20:18:37 GMT",
+        "expires": "-1",
+        "nel": "{\"report_to\":\"network-errors\",\"max_age\":86400,\"success_fraction\":0.001,\"failure_fraction\":1.0}",
+        "p3p": "CP=\"DSP CUR OTPi IND OTRi ONL FIN\"",
+        "pragma": "no-cache",
+        "referrer-policy": "strict-origin-when-cross-origin",
+        "report-to": "{\"group\":\"network-errors\",\"max_age\":86400,\"endpoints\":[{\"url\":\"https://identity.nel.measure.office.net/api/report?catId=GW+estsfd+san\"}]}",
+        "strict-transport-security": "max-age=31536000; includeSubDomains",
+        "x-content-type-options": "nosniff",
+        "x-ms-ests-server": "2.1.11829.9 - SCUS ProdSlices",
+        "x-ms-request-id": "59b1eb01-764c-46df-9381-4b82fbd73600"
+      }
+    },
+    {
+      "method": "POST",
+      "url": "https://azure_managedhsm.managedhsm.azure.net/rng",
+      "query": {
+        "api-version": "7.3"
+      },
+      "requestBody": "{\"count\":128}",
+      "status": 200,
+      "response": "{\"value\":\"mqFaqdp48lvoPR7mEbcRTb5y5C7ZdQRCCMLa7EyatllLmFlzaH0CNZ1AOGrVQAoubp5mRctUlZEog4mEIqjxA6D0vl1fR43Qs-ZrDpf6nHyUuRXt7gNdobWRPtHxYOCJucBwK-M_X3H7CBaSdD2pi-1B-BVxBxnYtHkBtCdcpJo\"}",
```

Comment:
Is the response value here sanitized?

Reply:
It's not sanitized but it's safe, it's just random bytes returned from the "get random bytes" endpoint